pkg/controller/operators/catalogtemplate/operator.go

```diff
@@ -130,6 +130,11 @@ func (o *Operator) syncCatalogSources(obj interface{}) error {
 		return nil
 	}
 
+	// This catalog template feature is not usable in configmap/internal catalogsource
+	if inputCatalogSource.Spec.SourceType == v1alpha1.SourceTypeInternal || inputCatalogSource.Spec.SourceType == v1alpha1.SourceTypeConfigmap {
```

Comment:
Any value in adding a debug-level log that communicates this catalogsource was skipped as it was non-grpc-based?

Reply:
Also, this implementation seems reasonable the short term, but we'd likely want to handle this properly at the caching-level and propagate the source type as a label (like we had talked about offline) so it's label selectable. Maybe add a TODO comment for this so that doesn't get lost?